examples/big_models_with_accelerate/cpu_offloading_fp8.py

```diff
@@ -24,3 +24,4 @@
     tokenizer=AutoTokenizer.from_pretrained(MODEL_ID),
     output_dir=OUTPUT_DIR,
 )
+model.save_pretrained(OUTPUT_DIR)
```

Comment:
Why save twice?

Reply:
Where do you see it being saved twice?

Reply:
Did more reading, the model is not saved twice but you pass the output_dir to oneshot and to save_pretrained which seems misleading

Reply:
This is true of most of the examples. We shouldn't pass the outputdir to oneshot if it's not going to be used (we should even trigger a warning/error for this)

Reply:
yeah the saving twice is confusing and part of this PR is to remove it in from oneshot. I think we should also remove the argument from oneshot.